lib/ansible/modules/database/postgresql/postgresql_copy.py

```diff
@@ -0,0 +1,427 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_copy
+short_description: Copy data between a file and a table
+description:
+- Copy data between a file and a table U(https://www.postgresql.org/docs/current/sql-copy.html).
+version_added: '2.9'
+
+options:
+  copy_to:
+    description:
+    - Copy the contents of a table to a file.
```

Comment:
would be great to mention that copy from/to file requires superuser privilege

Reply:
Great! thank you)